src/surfaces/cardActionArea/cardActionArea.tsx

```diff
@@ -0,0 +1,17 @@
+import MUICardActionArea from '@material-ui/core/CardActionArea';
+import React from 'react';
+
+function CardActionArea(props) {
+    const {
+        ...otherProps
+    } = props;
+
+    return (
+        <MUICardActionArea
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...otherProps}
+        />
+    );
```

Comment:
Couldn't we skip destructuring `props` into `otherProps` and just just do this?
```JavaScript
return (
    <MUICardActionArea
        // eslint-disable-next-line react/jsx-props-no-spreading
        {...props}
    />
);

Reply:
Fixed.

Reply:
Looks good.  Resolving.